e2e/tests/transfer/base_test.go

```diff
@@ -103,6 +103,10 @@ func (s *TransferTestSuite) TestMsgTransfer_Succeeds_Nonincentivized() {
 
 		expected := testvalues.IBCTransferAmount
 		s.Require().Equal(expected, actualBalance.Int64())
+
+		if testvalues.HumanReadableDenomFeatureReleases.IsSupported(chainBVersion) {
+			s.AssertHumanReadableDenom(ctx, chainB, chainADenom, channelA)
```

Comment:
cc @crodriguezvega

Reply:
I put together both assertions (check metadata is stored, check that balance returns the human readable denom). I hope that's ok, @chatton.